client/my-sites/upgrades/domain-management/name-servers/dns-templates.jsx

```diff
@@ -2,23 +2,53 @@
  * External dependencies
  */
 import React, { Component } from 'react';
-import { find } from 'lodash';
+import { find, replace } from 'lodash';
 import { localize } from 'i18n-calypso';
 
 /**
  * Internal dependencies
  */
+import ButtonGroup from 'components/button-group';
 import Card from 'components/card';
+import { dnsTemplates } from 'lib/domains/constants';
 import DnsTemplateButton from './dns-template-button';
-import Office365 from '../dns/office-365';
+import EmailProvider from '../dns/email-provider';
 
 class DnsTemplates extends Component {
 	constructor( props ) {
 		super( props );
+		const { translate } = this.props;
 		this.state = {
 			currentComponentName: null,
+			currentProviderCardName: null,
 			templates: [
-				{ name: 'Office 365', Component: Office365 }
+				{
+					name: 'G Suite',
+					label: translate( 'G Suite Verification Token - from the TXT record verification' ),
```

Comment:
@klimeryk Since both are known to be products that won't be translated, a placeholder can be used! Please also add a translator comment that will explain what the placeholder values can be.